test/powershell/Modules/Microsoft.PowerShell.Utility/ConvertFrom-StringData.Tests.ps1

```diff
@@ -89,14 +89,17 @@ a:b
         { $sampleData | ConvertFrom-StringData -Delimiter ':' } | Should -Not -Throw
     }
 
-    It 'is able to parse <StringData> with delimiter "<Delimiter>"' -TestCases $TestCases {
+    It 'is able to parse <StringData> with delimiter "<Delimiter> with <stringdata>"' -TestCases $TestCases {
         param($Delimiter, $StringData, $ExpectedResult)
 
         $Result = ConvertFrom-StringData -StringData $StringData -Delimiter $Delimiter
 
-        foreach ($Key in $ExpectedResult.Keys) {
-            $Key | Should -BeIn $ExpectedResult.Keys
-            $Result.$Key | Should -Be $ExpectedResult.$Key
-        }
+        $key = $ExpectedResult.Keys
```

Comment:
Should it be:
```suggestion
        $key = $Result.Keys
```

Reply:
We expect the result to match the expected hashtable, so I'm using the key from the expected hashtable to retrieve the value from the result (line 103).  In either case, due to the fact there is only one entry in the hashtable, either line would work the same if it passes or if it fails since we check the key values on line 100.

Reply:
The test looks tricky :-)